tracer/src/Datadog.Trace/AppSec/Security.cs

```diff
@@ -442,6 +442,15 @@ private void InitWafAndInstrumentations(bool fromRemoteConfig = false)
                 {
                     _configurationStatus.FallbackEmbeddedRuleSet ??= RuleSet.From(_wafInitResult.EmbeddedRules);
                 }
+
+                if (!fromRemoteConfig)
+                {
+                    // occurs the first time we have initialize the WAF
+                    TelemetryFactory.Metrics.SetWafVersion(_waf!.Version);
+                    TelemetryFactory.Metrics.RecordCountWafInit();
+                }
+
+                TelemetryFactory.Metrics.RecordCountWafUpdates();
```

Comment:
shouldnt this metric be line 206? we dont necessarily call `InitWafAndInstrumentations` when the waf is updated, only when ASM has been disabled and re enabled. And the [RFC ](https://docs.google.com/document/d/17dYHulDMa9XlZFWBPd4bAdFF-V1fU_0CYDLLMzcgilE/edit#heading=h.jq39arln8xio) seems to state "This metric is emitted when rules are updated (or reloaded) by the WAF (after a rule update coming from remote configuration for instance) so maybe it should only be sent when `configurationStatus` contains the key `rules` but I'm not sure if it's still valid as the RFC was written some time ago and the waf now always calls `Update` for any change...